packages/quick-edit/web/cloudflare-logic/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+	"compilerOptions": {
+		"module": "commonjs",
+		"target": "ES2020",
+		"outDir": "dist",
```

Comment:
I saw all the `dist` changed to `web` however this this still `dist` I think I am missing something lol